tests/headers/complex_global.h

```diff
@@ -0,0 +1 @@
+double _Complex globalValue;
```

Comment:
You can probably stick that in `tests/headers/complex.h` if you prefer, though it's fine as is too :)